src/frontend/screens/Achievements/GameAchievementDetails/index.tsx

```diff
@@ -0,0 +1,139 @@
+import { GameAchievements } from '@hyperplay/ui'
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { achievementsSortOptions } from '..'
+import walletState from 'frontend/state/WalletState'
+import { useTranslation } from 'react-i18next'
+import { observer } from 'mobx-react-lite'
+import AchievementStoreState from 'frontend/state/AchievementStoreState'
+import MintAchievementsState from 'frontend/state/MintAchievementsState'
+
+const pageSize = 6
+
+function isTimestampInPast(unixTimestamp: number) {
+  const currentTime = new Date().getTime()
+  const timestampInMilliseconds = unixTimestamp * 1000 // Convert to milliseconds
+
+  return timestampInMilliseconds < currentTime
+}
+
+export default observer(function GameAchievementDetails(): JSX.Element {
+  const { t } = useTranslation()
+  const { id } = useParams()
+
+  const [selectedSort, setSelectedSort] = useState(achievementsSortOptions[0])
+
+  const achievementsToBeMinted = MintAchievementsState.achievementsToBeMinted
+  const isLoading = MintAchievementsState.isLoading
+  const handleMint = MintAchievementsState.handleMint
+  const handleUpdate = MintAchievementsState.handleUpdate
+  const achievementsToBeUpdated = MintAchievementsState.achievementsToBeUpdated
+
+  const numFreeMints = AchievementStoreState.numFreeMints
+  const individualAchievements = AchievementStoreState.individualAchievements
+  const summaryAchievement = AchievementStoreState.summaryAchievements.data[0]
+
+  useEffect(() => {
+    AchievementStoreState.getSummaryAchievements({
+      page: 1,
+      pageSize,
+      sort: selectedSort.value,
+      filter: 'ALL'
+    })
+    AchievementStoreState.getIndividualAchievements({
+      gameId: id as string,
+      page: 1,
+      pageSize,
+      sort: selectedSort.value
+    })
+  }, [])
```

Comment:
We need this once or per `id` and `selectedSort.value`

Reply:
this component has changed significantly since this comment and I'm not sure if this is still relevant

Reply:
No longer seems 🙌